sections/appendix/standard_dev.tex

```diff
@@ -0,0 +1,11 @@
+\subsection{\gls{lda} standard deviation}\label{sec:std_dev}
+Since the \gls{lda} model uses a generative process, we see how confident the standard model is, in the resulting topic coherence.
+We have run 10 models using the hyper parameters $K=90$, $\alpha=0.01$ and $\beta=0.1$.
+Measuring the confidence of our Gibbs sampler can be calculated by the standard deviation by using the results from the 10 models' topic coherence.
+
+We calculate the standard deviation by:
+\begin{equation}
+	SD = \sqrt{\frac{\sum_{i=1}^{n}(x_i-\bar{x})^2}{n-1}}

```

Comment:
Jeg kan se at formlen er en smule anderledes afhængig af om man ser sin data som en fuld population eller et sample af en population. Jeg ville nok tage vores 10 resultater som en population, og ændre formlen til det, men hvis i har et argument for at tænke på det som et sample kan vi også gå med det.

Reply:
jeg var lidt i tvilv om hvilken en det var :)

Reply:
Jeg er personligt heller ikke sikker på hvad er rigtigt, for jeg kunne ikke finde et tydeligt svar på hvad gælder for os. Hvis vi bruger population versionen skal ligningen lige ændres, ellers er jeg tilfreds.